members.html

```diff
@@ -262,6 +262,38 @@ <h1>PhD - 3rd year</h1>
 
     </table>        
     
+  <h1>PhD - 4th year</h1>
+
+    <table>
+        {% for item in site.data.members.phd_forth %}
```

Comment:
Fix typo here as well so the code continues to work:
```suggestion
        {% for item in site.data.members.phd_fourth %}
```